Lib/fontbakery/specifications/adobe_fonts.py

```diff
@@ -106,6 +109,21 @@ def com_adobe_fonts_check_name_empty_records(ttFont):
         yield PASS, ("No empty name table records found.")
 
 
+@check(
+    id='com.adobe.fonts/check/consistent_upm'
```

Comment:
it would be good to add here a `rationale=` attribute with an overview of the reasoning behind this check.